### DISCUSSION TITLE
Model Selector Dropdown Ordering Not Working v0.4.1 (latest)

### DISCUSSION THREAD

**Discussion by sascha-cariad** [open-webui/open-webui#7086] [Issues]
That is not possible anymore with the latest version (v0.4.1):
```
Model Selector Dropdown Ordering: 
Models can be effortlessly organized by dragging and dropping them into desired positions 
within the model workspace, which will then reflect the changes in the model dropdown menu.
```
### /workspace/models
<img width="562" alt="image" src="https://github.com/user-attachments/assets/421e7461-9d09-47aa-891a-a8a35c78f170">

### /admin/settings -> Models
<img width="801" alt="image" src="https://github.com/user-attachments/assets/f73b83cb-bee8-4d17-a460-3557c6d83428">

**Comment by rangerrick337** [comment]
Similarly cannot flag them as favorites and pin them to the top of the list anymore.

**Comment by ticpu** [comment]
This is quite a pain for users who don't know model names, I used to put the most useful one on top.

**Reply by ticpu** [reply]
v0.4.5 fixes this
🎨 Model Order/Defaults Reintroduced

**Comment by mowtschan** [comment]
Looks like the maintainer already aware about the problem, here the issue: #7011
